server.js

```diff
@@ -1,15 +1,45 @@
-'use strict';
+const express = require("express");
+const dotenv = require("dotenv"); 
+dotenv.config();
 
-const express = require('express');
+const jexiaSDK = require("jexia-sdk-js/node");
+const dataModule = jexiaSDK.dataOperations();
+const field = jexiaSDK.field();
 
-// Constants
-const PORT = 80;
+const credentials = {
+  projectID: process.env.PROJECT_ID,
+  key: process.env.API_KEY,
+  secret:process.env.API_SECRET
+};
+
+jexiaSDK.jexiaClient().init( credentials,dataModule);
+const todo = dataModule.dataset(process.env.DATASET);
+
+// if you want to run locally change port to 3000
+// for Apphosting use port 80
+const PORT = 80; 
+
+// to deploy it to AppHosting please use IP:0.0.0.0
 const HOST = '0.0.0.0';
 
 // App
 const app = express();
 app.get('/', (req, res) => {
-  res.send('Hello world\n');
+  todo
+  .select()
+  .limit(200)
+  .fields("text", "id","done")
+  .where(field=> field("text").isLike("%4").and(field("id").isNotNull()))
```

Comment:
> isLike("%4")

Are you filtering todo items? Is this for (local) testing purposes or intended?

Reply:
Intend, to show example of how to use isLike operator. For me, it was not obvious